modules/flowable-engine/src/test/java/org/flowable/standalone/initialization/ProcessEngineInitializationTest.java

```diff
@@ -73,18 +74,15 @@ public void testVersionMismatch() {
             sqlSession.close();
         }
 
-        try {
-            // now we can see what happens if when a process engine is being
-            // build with a version mismatch between library and db tables
-            ProcessEngineConfiguration.createProcessEngineConfigurationFromResource("org/flowable/standalone/initialization/notables.flowable.cfg.xml")
-                    .setDatabaseSchemaUpdate(ProcessEngineConfiguration.DB_SCHEMA_UPDATE_FALSE).buildProcessEngine();
-
-            fail("expected exception");
-        } catch (FlowableWrongDbException e) {
-            assertTextPresent("version mismatch", e.getMessage());
-            assertEquals("25.7", e.getDbVersion());
-            assertEquals(ProcessEngine.VERSION, e.getLibraryVersion());
-        }
+        // now we can see what happens if when a process engine is being
+        // build with a version mismatch between library and db tables
+        assertThatThrownBy(() -> ProcessEngineConfiguration
+                .createProcessEngineConfigurationFromResource("org/flowable/standalone/initialization/notables.flowable.cfg.xml")
+                .setDatabaseSchemaUpdate(ProcessEngineConfiguration.DB_SCHEMA_UPDATE_FALSE).buildProcessEngine())
+                .isExactlyInstanceOf(FlowableWrongDbException.class)
```

Comment:
Use `isInstanceOfSatisfying` so we can test the `FlowableWrongDbException#getDbVersion` and `FlowableWrongDbException#getLibraryVersion`, not just that the message has it.

Reply:
Done; thanks.